tasks/request_status_for_granule/request_status_for_granule.py

```diff
@@ -47,34 +51,32 @@ def __init__(self, job_id: str, granule_id: str):
 
 
 def task(
-    granule_id: str, db_connect_info: Dict, job_id: str = None
+    collection_id: str, granule_id: str, db_connect_info: Dict, job_id: str = None
 ) -> Dict[str, Any]:
     # noinspection SpellCheckingInspection
     """
     Args:
-        granule_id: The unique ID of the granule to retrieve status for.
+        collection_id: The ID of the collection containing the granule.
+        granule_id: The ID of the granule to retrieve status for.
         db_connect_info: The {database}.py defined db_connect_info.
         job_id: An optional additional filter to get a specific job's entry.
     Returns: See output.json
     """
-    if granule_id is None or len(granule_id) == 0:
-        raise ValueError("granule_id must be set to a non-empty value.")
```

Comment:
I know I pushed for these checks initially, but it's causing a lot of code clutter and maintenance. Recommend we go back to trusting the schema checks.